tdex/hooks/pre-start

```diff
@@ -6,8 +6,9 @@ if [[ -f "${HIDDEN_SERVICE_FILE}" ]]; then
 	exit
 fi
 
-"${UMBREL_ROOT}/scripts/app" compose "${APP_ID}" up --detach tdexd
 "${UMBREL_ROOT}/scripts/app" compose "${APP_ID}" up --detach tor
+"${UMBREL_ROOT}/scripts/app" compose "${APP_ID}" up --detach oceand
+"${UMBREL_ROOT}/scripts/app" compose "${APP_ID}" up --detach tdexd
```

Comment:
I think you can revert this change. The `oceand` container (as well as the dashboard) will be brought up after this script runs. This is just making sure that a tor hidden service is created for the daemon.

Reply:
I made the change because `tdexd` requires an already running `oceand` to start correctly. 
In light of this, do you think I should still revert the changes?

Reply:
ah I see, sorry for misunderstanding. I see now that you have tdex depending on oceand in the compose file. Great work catching this!